setup.cfg

```diff
@@ -0,0 +1,21 @@
+[bumpversion]
+current_version = 0.1.0
+commit = True
+tag = True
+
+[bumpversion:file:setup.py]
+search = version='{current_version}'
+replace = version='{new_version}'
+
+[bumpversion:file:cidc_schemas/__init__.py]
+search = __version__ = '{current_version}'
+replace = __version__ = '{new_version}'
+
+[bdist_wheel]
+universal = 1
+
+[flake8]
+exclude = docs
+
+[aliases]
+# Define setup.py command aliases here
```

Comment:
Looks like this was copied verbatim from here https://github.com/CIMAC-CIDC/cidc-schemas/blob/master/setup.cfg

Not sure what that file's left over from, but we don't currently use the [bumpversion](https://github.com/peritus/bumpversion) package in any of our repos, so I wouldn't add this config file here (and we should probably remove it from the cidc schemas repo, too).